File: goormlevel/355872/1. 흰색 칸의 체스말 개수/흰색 칸의 체스말 개수.py
import sys

def solve():
    input = sys.stdin.readline
    write = sys.stdout.write

    chessmap = [input().rstrip() for _ in range(8)]

    answer = sum(1 for r in range(8) for c in range(8) if (r + c) % 2 == 0 and chessmap[r][c] == '1')

    write(str(answer) + "\n")

if __name__ == "__main__":
    solve()

File: goormlevel/355872/1. 흰색 칸의 체스말 개수/흰색 칸의 체스말 개수.c
#include <stdio.h>

int main() {
	char chessmap[8][10];
	for (int i = 0; i < 8; i++) {
		if (scanf("%s", &chessmap[i]) != 1) return 0;
	}

	int answer = 0;
	for (int i = 0; i < 8; i++) {
		for (int j = 0; j < 8; j++) {
			if ((i + j) % 2 == 0 && chessmap[i][j] == '1') {
				answer++;
			}
		}
	}

	printf("%d\n", answer);

	return 0;
}
